Add deprecated status (#9)
3rdparty/stout-grpc-examples: README.md

[![status: archive](https://github.com/GIScience/badges/raw/master/status/archive.svg)](https://github.com/GIScience/badges#archive) [![No Maintenance Intended](http://unmaintained.tech/badge.svg)](http://unmaintained.tech/)

# DEPRECATED - migrated into [`3rdparty/eventuals/test/grpc/route_guide`](https://github.com/onelxj/eventuals/tree/routeGuideTest/test/grpc/route_guide).

> ## NOTE: [`eventuals-tutorials`](https://github.com/3rdparty/eventuals-tutorial) is also a good place to see how you can start the project using [`eventuals`](https://github.com/3rdparty/eventuals).

Reviewer: ```suggestion
## NOTE: [`eventuals-tutorials`](https://github.com/3rdparty/eventuals-tutorial) is also a good place to see how you can start your own project that uses [`eventuals`](https://github.com/3rdparty/eventuals).
```
